models/engine/db_storage.py

```diff
@@ -74,3 +74,14 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """object based on the class name and its ID, or None"""
+        objs = self.all(cls)
+        if cls in classes and id:
+            return objs.get("{}.{}".format(cls, id))
+        return None
+
+    def count(self, cls=None):
+        """count the number of objects"""
+        return (len(self.all(cls)))
```

Comment:
- Your count function its very efficient but making it a one-liner makes it lose readability:

> You should save the consulted objects in a variable and return the size of that consulted object dictionary